jax/lax/lax.py

```diff
@@ -400,6 +400,7 @@ def concatenate(operands, dimension):
                             operand_shapes=tuple(o.shape for o in operands))
 
 Precision = xla_client.PrecisionConfig.Precision
+Precision.__repr__ = lambda precision: precision.name
```

Comment:
nit: this would be a better __str__ than __repr__, but if it makes the test too painful feel free to keep as-is.

Reply:
You're right; fixed